core/engine/src/value/tests.rs

```diff
@@ -636,7 +636,7 @@ fn to_int32() {
         check_to_int32!(-3_000_000_000.5 => 1_294_967_296);
         check_to_int32!(-3_000_000_000.75 => 1_294_967_296);
 
-        let base = 2f64.powi(64);
+        let base = 18_446_744_073_709_552_000.0; // approx. 2^64
```

Comment:
Did this fix anything?

Reply:
Yes, the Miri VM doesn't have the exact same semantics for f64 than the host machine in complex operations like `powi` or `sqrt`, so it was causing failures. Using the exact value that we want is a lot more reliable.